protocol/proofs/fdg-refund-mode.md

```diff
@@ -0,0 +1,95 @@
+# FaultDisputeGame Refund Mode
+
+## Context
+
+Dealing with `FaultDisputeGame` bonds appropriately is an important part of the current OP Stack
+incident response process. `FaultDisputeGame` stores player bonds inside of the `DelayedWETH`
+contract which imposes a 7 day delay on bond withdrawals. During this delay period, the 2/2 Safe
+managed jointly by the Security Council and the Optimism Foundation can hold bonds from specific
+games or from all games at the same time. Held bonds are transferred to this 2/2 Safe where they
+can be manually disbursed to the correct recipients.
+
+## Problem Statement
+
+Although `DelayedWETH` is an effective tool at preventing a buggy or malicious `FaultDisputeGame`
+from distributing bonds to a potential attacker, the actual process of returning those bonds back
+to the correct players is underspecified. Underspecified proceses can introduce significant stress
+to already stressful situations where executive decisions around fallbacks must be made quickly and
+confidently. We therefore want to introduce alternative low-stress options for bond recovery.
+
+## Proposed Solution
+
+We propose modifying the `FaultDisputeGame` to enter a "refund mode" if the game is blacklisted.
+When the `FaultDisputeGame` is in this refund mode it will distribute bonds back to the users who
+originally deposited those bonds.
+
+### Preventing Complex States
+
+`FaultDisputeGame` currently allows users to begin withdrawing bonds before the game has fully
+resolved. It would likely be necessary to modify the `FaultDisputeGame` to block withdrawals until
+full resolution so that we can more easily reason about the security properties of this change. It
+may otherwise be possible to be caught in strange states.
+
+For instance, `FaultDisputeGame` instances are usually expected to be blacklisted only after they
+resolve incorrectly, so it may be possible for some users to begin withdrawing *before* a game is
+blacklisted, then the game becomes blacklisted and those same users can withdraw their original
+bonds. We can avoid this sort of double-counting and generally simplify the security model of the
+contract by preventing bond withdrawals until the game has resolved.
```

Comment:
I like the idea of simplifying when bonds can be claimed. Having it match when the dispute game could be used withdrawals (ie 3.5 days after the game finalizes) makes a lot of sense and is much easier to reason about.